src/main/scala/com.snowplowanalytics/sqs2kinesis/EventsStreamModule.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+package com.snowplowanalytics.sqs2kinesis
+
+import com.github.matsluni.akkahttpspi.AkkaHttpClient
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.Message
+import com.amazonaws.services.kinesis.AmazonKinesisAsyncClientBuilder
+import com.amazonaws.services.kinesis.model.PutRecordsResultEntry
+import com.amazonaws.services.kinesis.model.PutRecordsRequestEntry
+import java.nio.ByteBuffer
+import akka.actor.ActorSystem
+import akka.stream.scaladsl._
+import akka.stream.alpakka.sqs.scaladsl.SqsSource
+import akka.stream.alpakka.sqs.SqsSourceSettings
+import akka.stream.alpakka.kinesis.KinesisFlowSettings
+import akka.stream.alpakka.kinesis.scaladsl.KinesisFlow
+import akka.stream.alpakka.sqs.MessageAction
+import akka.stream.alpakka.sqs.scaladsl.SqsAckSink
+import akka.NotUsed
+import akka.stream.ActorAttributes
+import akka.stream.Supervision
+import scala.concurrent.duration._
+import com.typesafe.scalalogging.Logger
+
+object EventsStreamModule {
+
+  case class StreamConfig(
+    sqsQueue: String,
+    sqsKeyValueSeparator: Char,
+    kinesisStreamName: String
+  )
+
+  val logger = Logger[EventsStreamModule.type]
+
+  type KinesisKeyAndMsg = (String, ByteBuffer)
+
+  def sqsMsg2kinesisMsg(sqsKeyValueSeparator: Char): String => KinesisKeyAndMsg =
+    msgBody => {
```

Comment:
```suggestion
    sqsMsg => {
```
would ease the understanding IMHO